tests/core_tests.py

```diff
@@ -114,6 +114,34 @@ def test_misc(self):
         assert self.client.get('/health').data.decode('utf-8') == "OK"
         assert self.client.get('/ping').data.decode('utf-8') == "OK"
 
+    def test_testconn(self):
+        data = json.dumps({'uri': 'sqlite:////tmp/caravel_unittests.db'})
+        response = self.client.post('/caravel/testconn', data=data, content_type='application/json')
+        assert response.status_code == 200
+
+        database = (
```

Comment:
This test has three parts, but now that I think of it, it should really just use the last two since some of the test cases are against a sqlite db and others use mysql or postgres.  So I think I should remove the hard-coded uri here and just rely on the other two tests.  One of those two tests with the password masked, the other with the password included in the uri.